nsapass

```diff
@@ -305,11 +320,13 @@ def nsa_undel(name, index):
     d = load(lock=True)
     password = d[section][name][i][KEY_PASS]
     note = d[section][name][i][KEY_NOTE]
-    nsa_add(name, password, note, d=d, save_d=False)
+    nsa_add(SECTION_ENTRIES, name, password, note, d=d, save_d=False)
     nsa_del(name, index, d=d, save_d=False)
     save(d)
 
-def nsa_mod(name, index, new_name, new_pass, new_note):
+def nsa_mod(name, index, new_name, new_pass, prompt_new_pass, new_note):
+    if prompt_new_pass:
+        new_pass = pass_prompt()
```

Comment:
what should happen if both `-p` and `-P` are given?

Reply:
currently `-P` ends up prompting for a password until it gets one that overwrites the password given by `-p password123`.

i think password obtained from `-P` is probably more private, since it is guaranteed to not live in the history file of the shell, which is why i preferred it for now.

but now that you made me think more, i guess it might be better to report and error and exit.  so that the user who puts a useless `-p password123` learns that it's useless and stops doing it (to ensure that his pass doesn't enter his shell's history file).

better thoughts?

Reply:
Ah, I should've written that making this an error would be my preference (since the user's intentions are unclear)